tests/server/signupRoute.test.js

```diff
@@ -0,0 +1,71 @@
+import { db } from "../../server/db";
+import Supertest from "supertest";
+
+const App = require("../../server/index");
+const request = Supertest(App);
+
+jest.mock("../../server/db");
+
+const noUNOrPass = "Error: Username or password not set!";
+
+describe("Test signup route", () => {
```

Comment:
add 
```
beforeAll(() => {
    jest.spyOn(console, "log").mockImplementation(() => {});
    jest.spyOn(console, "error").mockImplementation(() => {});
  });
```
to stop the test from logging to the console

Reply:
Resolved in [89c391e](https://github.com/PistolRcks/web-puzzle-project/pull/24/commits/89c391e96c30d02d5b83bb1f35071283e25411ef).